bin/carthage-bootstrap-if-needed

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+if ! diff Cartfile.resolved Carthage/Cartfile.resolved &>/dev/null; then
+  carthage bootstrap 
+  cp Cartfile.resolved Carthage
+fi
```

Comment:
Do you know what command CircleCI uses for git submodules? On Travis it didn't do `--recursive` or something like that so I had to disable the built in support and then call it manually.